src/cargo/ops/cargo_rustc/output_depinfo.rs

```diff
@@ -70,7 +70,19 @@ pub fn output_depinfo<'a, 'b>(context: &mut Context<'a, 'b>, unit: &Unit<'a>) ->
     let mut deps = HashSet::new();
     let mut visited = HashSet::new();
     let success = add_deps_for_unit(&mut deps, context, unit, &mut visited).is_ok();
-    let basedir = None; // TODO
+    let basedir_string;
+    let basedir = match context.config.get_string("dep-info-basedir") {
```

Comment:
I think here we'll want a `?` at the end to propagate errors about misconfiguration (aka used an integer instead of a string)

Reply:
Also, I wonder if this should perhaps be `build.dep-info-basedir` as well?

Reply:
I'll add the doc.

I will return the error instead of ignoring it in that case.

I have no preference on where to put the config so if you like `build.dep-info-basedir` I love it.

Reply:
Ok sounds good! Let's use `build.dep-info-basedir` and I think this should then be good for r+